pkg/services/alerting/notifiers/discord.go

```diff
@@ -24,29 +24,43 @@ func init() {
 		Factory:     NewDiscordNotifier,
 		OptionsTemplate: `
       <h3 class="page-heading">Discord settings</h3>
-      <div class="gf-form">
-        <span class="gf-form-label width-14">Webhook URL</span>
-        <input type="text" required class="gf-form-input max-width-22" ng-model="ctrl.model.settings.url" placeholder="Discord webhook URL"></input>
+      <div class="gf-form max-width-30">
+        <span class="gf-form-label width-6">Mention</span>
+        <input type="text"
+          class="gf-form-input max-width-30"
+          ng-model="ctrl.model.settings.mention"
```

Comment:
Since the name of the field in discord is named content I think this property should be called content as well. Aligning the alert notifier more with discord. The help text can suggest that users can be mentioned in the content field.

Reply:
Nice idea. I'll update the field label too, to say "Message content" instead of "Mention", and help text will say about adding mentions here.